README.md

```diff
@@ -0,0 +1,16 @@
+![GitHub last commit](https://img.shields.io/github/last-commit/sdonkov/dawg)
+
+# DAWG
+
+Try to implement DAWG (Directed Acyclic Word Graph) also known as DAFSA is a library for computing Deterministic Acyclic Finite State Automata.
```

Comment:
This is a really weird sentence, perhaps removing "Try to implement" will improve it...